pkgs/development/libraries/SDL/default.nix

```diff
@@ -25,13 +25,14 @@ stdenv.mkDerivation rec {
 
   # Since `libpulse*.la' contain `-lgdbm', PulseAudio must be propagated.
   propagatedNativeBuildInputs =
```

Comment:
`mesa` and `alsa` shouldn't be native, neither should be libcap, that's an err on my side. It should be Linux-only.